go.mod

```diff
@@ -37,3 +37,5 @@ require (
 	google.golang.org/appengine v1.0.0 // indirect
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
 )
+
+replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
```

Comment:
Could we also change all references from `github.com/keybase/go-keychain` to `github.com/99designs/go-keychain`, but this indicates we expect the replacement to be temporary?

Reply:
I think it's trickier than that because it's an indirect dependency, via 99designs/keyring; I think we'd have to vendor that and patch its imports too. I guess this is what `replace` was made for; I've never used it before, I just cargo-culted @mtibben's comment advice ;)